package.json

```diff
@@ -56,8 +56,9 @@
     "jstz": "~1.0.7",
     "lodash": "~4.15.0",
     "multiselect": "~0.9.12",
+    "prop-types": "^15.5.9",
     "react": "^15.1.0",
```

Comment:
in our production builds, we still use 15.3 (see https://github.com/theforeman/foreman-packaging/tree/rpm/develop/nodejs-react) if you are OK with that version I'll ack.